scripts/wikipedia_Scraper/wikipediaBS4.py

```diff
@@ -0,0 +1,34 @@
+import requests # API for interacting with internet
+from bs4 import BeautifulSoup # web scraper
+
+def switch(answer):
```

Comment:
This works but we could refactor this to be:

```
if answer.lower() == "yes":
    answer = "y"
elif answer.lower() == "no":
    answer = "n"
else:
    # print"invalid choice" [...]
match answer:
    case "y":
        return True
     case "n":
        return False
```

This would also cover the case where the input is for example "12" or any other non valid input.

Reply:
Overall I love that you created two different versions, the only issue at the moment is what happens if we do not pass "y" "n" "yes" or "no" as input. everything else looks good to me!